01-starting-with-data.Rmd

```diff
@@ -330,6 +330,25 @@ We'll learn why this is so in the next lesson.
 > (c) `max(dat[5, 3:7])`
 > (d) `max(dat[5, 3, 7])`
 
+
+> ## Challenge - Slicing and re-assignment {.challenge}
+> 
+> Let's pretend there was something wrong with the instrument on the first five days for every second patient (#2, 4, 6, etc.), which resulted in the measurements being twice as large as they should be.
```

Comment:
I think you should mention something like "To follow up from the previous data frame" to ensure that they know the question is related to the previous one.  Otherwise, they might make the vector as instructed in the first point, and they might get confused that they were supposed to turn it into a data frame afterwards.